x-pack/plugins/enterprise_search/public/applications/app_search/components/api_logs/components/empty_state.test.tsx

```diff
@@ -21,7 +23,7 @@ describe('EmptyState', () => {
 
     expect(wrapper.find('h2').text()).toEqual('No API events in the last 24 hours');
     expect(wrapper.find(EuiButton).prop('href')).toEqual(
-      expect.stringContaining('/api-reference.html')
+      expect.stringContaining(`${docLinks.appSearchApis}`)
```

Comment:
```suggestion
      expect.stringContaining(docLinks.appSearchApis)
```
No need for a template literal here